assembl/models/idea_msg_columns.py

```diff
@@ -129,5 +124,52 @@ def ensure_ordering(cls):
         for idea in ideas:
             cls.ensure_ordering_for_idea(idea)
 
+    def get_column_synthesis(self):
+        synthesis = self.db.query(ColumnSynthesisPost
+            ).join(ColumnSynthesisPost.idea_links_of_content
+            ).filter(
+                IdeaContentLink.idea_id == self.idea_id,
+                ColumnSynthesisPost.tombstone_date == None,
+                Content.message_classifier == self.message_classifier
+            ).all()
```

Comment:
Why not use `.first()`, which would return the synthesis or `None` ?

Reply:
I forgot about `.first()`, yeah you can probably use it here.

Reply:
done